extensions/kafka/deployment/src/main/java/org/apache/camel/quarkus/component/kafka/deployment/KafkaProcessor.java

```diff
@@ -68,4 +69,11 @@ public void configureKafkaComponentForDevServices(
             }
         }
     }
+
+    @BuildStep
+    public ReflectiveClassBuildItem reflectiveClasses() {
+        return new ReflectiveClassBuildItem(true, true,
```

Comment:
Do we need to enable reflection for methods ? or fields could be enough ?

Reply:
yeah, I think fields is enough currently.